libsolidity/formal/Predicate.cpp

```diff
@@ -50,20 +51,22 @@ Predicate const* Predicate::create(
 	return &m_predicates.emplace(
 		std::piecewise_construct,
 		std::forward_as_tuple(functorName),
-		std::forward_as_tuple(move(predicate), _type, _node, _contractContext)
+		std::forward_as_tuple(move(predicate), _type, _node, _contractContext, move(_scopeStack))
```

Comment:
You can't get rid of `std::` here?

Reply:
we could, but in this case I'd prefer to have it explicit